model/database/cosmosdb-metrics.yaml

```diff
@@ -0,0 +1,57 @@
+groups:
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "Request charge consumed by the operation."
+    instrument: histogram
+    unit: "{request_units}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.operation.regions_contacted
+    type: metric
+    metric_name: db.cosmosdb.operation.regions_contacted
```

Comment:
is it a proxy to some other thing like number of retries or number of fallbacks? 
It's not immediately obvious how to interpret this metric and whether high values indicate a problem.

Reply:
this metrics tells, how many cross regional calls are there and help us to find a pattern, if high latency is due to these calls.

Reply:
removing this metrics as of now.